From 547af58a90ac0ec636e6dd97704b13bb4d2078c0 Mon Sep 17 00:00:00 2001
From: minimac <macmini@Soportes-Mac-mini.local>
Date: Thu, 5 Dec 2019 23:07:26 -0600
Subject: [PATCH] fix buttons in portrait mode

---
 ios/RSSignatureView.m | 33 ++++++++++-----------------------
 1 file changed, 10 insertions(+), 23 deletions(-)

diff --git a/ios/RSSignatureView.m b/ios/RSSignatureView.m
index 526d810..27b735d 100644
--- a/ios/RSSignatureView.m
+++ b/ios/RSSignatureView.m
@@ -69,7 +69,7 @@ - (void)layoutSubviews
 
 		[self addSubview:sign];
 
-		if ( UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad ) {
+		if ( UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad || [_viewMode  isEqual: @"portrait"]) {
 
 			if (_showTitleLabel) {
 				titleLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, self.bounds.size.width, 24)];
@@ -113,28 +113,15 @@ - (void)layoutSubviews
 		}
 		else {
 			if (_showTitleLabel) {
-				if ( [_viewMode  isEqual: @"portrait"] ) {
-					titleLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, self.bounds.size.width, 24)];
-					[titleLabel setCenter:CGPointMake(self.bounds.size.width/2, self.bounds.size.height - 120)];
-
-					[titleLabel setText:@"x_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _"];
-					[titleLabel setLineBreakMode:NSLineBreakByClipping];
-					[titleLabel setTextAlignment: NSTextAlignmentCenter];
-					[titleLabel setTextColor:[UIColor colorWithRed:200/255.f green:200/255.f blue:200/255.f alpha:1.f]];
-					//[titleLabel setBackgroundColor:[UIColor greenColor]];
-					[sign addSubview:titleLabel];
-				}
-				else {
-					titleLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, self.bounds.size.height - 80, 24)];
-					[titleLabel setCenter:CGPointMake(40, self.bounds.size.height/2)];
-					[titleLabel setTransform:CGAffineTransformMakeRotation(DEGREES_TO_RADIANS(90))];
-					[titleLabel setText:@"x_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _"];
-					[titleLabel setLineBreakMode:NSLineBreakByClipping];
-					[titleLabel setTextAlignment: NSTextAlignmentLeft];
-					[titleLabel setTextColor:[UIColor colorWithRed:200/255.f green:200/255.f blue:200/255.f alpha:1.f]];
-					//[titleLabel setBackgroundColor:[UIColor greenColor]];
-					[sign addSubview:titleLabel];
-				}
+				titleLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, self.bounds.size.height - 80, 24)];
+				[titleLabel setCenter:CGPointMake(40, self.bounds.size.height/2)];
+				[titleLabel setTransform:CGAffineTransformMakeRotation(DEGREES_TO_RADIANS(90))];
+				[titleLabel setText:@"x_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _"];
+				[titleLabel setLineBreakMode:NSLineBreakByClipping];
+				[titleLabel setTextAlignment: NSTextAlignmentLeft];
+				[titleLabel setTextColor:[UIColor colorWithRed:200/255.f green:200/255.f blue:200/255.f alpha:1.f]];
+				//[titleLabel setBackgroundColor:[UIColor greenColor]];
+				[sign addSubview:titleLabel];
 			}
 
 			if (_showNativeButtons) {
